server/routes/sessions.js

```diff
@@ -44,20 +44,86 @@ const sessions = [
         sport: "Soccer",
         joined: false,
         dateTime:" dayjs('2022-04-17T15:30')",
-        type: "outdoor,"
+        type: "outdoor"
```

Comment:
I classified them because we didn't decide what kind of filters we want to apply. It is just a basic example of how the filter functionality will work. Do you have other ideas of what kind of filters we should apply?

Reply:
If we add more properties, we would need to update a few other other components to allow the user to manipulate these new properties (like create session and more info). Maybe we can just classify by sport for now, and add new properties if we get our more important tickets done and have some time on our hands?